examples/motorkit_dc_test.py

```diff
@@ -27,13 +27,13 @@
     time.sleep(1)
 
     print("Speed up...")
-    for i in range(-100, 0):
+    for i in range(0, -100, -1):
         speed = i * 0.01
         kit.motor1.throttle = speed
         time.sleep(0.01)
 
     print("Slow down...")
-    for i in reversed(range(-100, 0)):
+    for i in range(-100, 0):
```

Comment:
Same.
Current:
```
>>> for i in reversed(range(-10, 0)):
...     print(i)
...
-1
-2
-3
-4
-5
-6
-7
-8
-9
-10
```
Proposed change:
```
>>> for i in range(-10, 0):
...     print(i)
...
-10
-9
-8
-7
-6
-5
-4
-3
-2
-1
>>>
```

Reply:
Yeah. Beginner Python mistake. Should be
`for i in range(-100, 1):`

![image](https://user-images.githubusercontent.com/29906257/52905858-2b04e380-31f5-11e9-9178-b73af6077127.png)